File: app/src/main/java/com/tutv/android/datasource/dto/SeriesFollowedDTO.java
package com.tutv.android.datasource.dto;

public class SeriesFollowedDTO {
    private int seriesId;

    public SeriesFollowedDTO() {}

    public SeriesFollowedDTO(int seriesId) {
        this.seriesId = seriesId;
    }

    public int getSeriesId() {
        return seriesId;
    }

    public void setSeriesId(int seriesId) {
        this.seriesId = seriesId;
    }
}

File: app/src/main/java/com/tutv/android/datasource/dto/SeriesFollowedDTO.kt
package com.tutv.android.datasource.dto

import org.mockito.ArgumentMatchers
import org.mockito.InOrder
import org.mockito.Mockito

class SeriesFollowedDTO {
    private var seriesId = 0

    constructor() {}
    constructor(seriesId: Int) {
        this.seriesId = seriesId
    }

    fun getSeriesId(): Int {
        return seriesId
    }

    fun setSeriesId(seriesId: Int) {
        this.seriesId = seriesId
    }
}